### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by Fiscalizando** [danielkrupinski/Osiris#2833] [General]
P for legit, but unfortunately don’t have rage features like doubletap, resolver and lua support. But i liked the other features. There are infinites ways to create a legit config with osiris, but you can’t use on your main since it’s detected. Try using in a smurf with vac bypass.

**Comment by uzdiiilekk** [comment]
There are no rage features because it’s legit cheat ..

**Reply by Fiscalizando** [reply]
ik, I’m just saying this cuz many people use osiris with semirage or an poor rage cfg

**Reply by linuxer9** [reply]
you don't need lua support it's open source
